blog/models.py

```diff
@@ -0,0 +1,26 @@
+"""Models for Blog app."""
+from django.conf import settings
+from django.db import models
+from django.utils import timezone
+
+
+class Post(models.Model):
+    """Create Post model for Blog app."""
```

Comment:
It's a class, not a function. So there MUSTN'T be any action here.

Reply:
I should write """Post model for Blog app."""?

Reply:
that'd work